ceremonies/2023/i1-cert.yaml

```diff
@@ -0,0 +1,26 @@
+ceremony-type: intermediate
+pkcs11:
+  module: /usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so
+  pin: 1234
+  signing-key-slot: 1307844626
+  signing-key-label: root-x2
+inputs:
+  issuer-certificate-path: ../2020/root-x2.cert.pem
+  public-key-path: ./int-i1.key.pem
+outputs:
+  certificate-path: ./int-i1.cert.pem
+certificate-profile:
+  signature-algorithm: ECDSAWithSHA384
+  common-name: (FAKE) I1
```

Comment:
How'd we settle on the prefix "I" for this throwaway intermediate?

Reply:
You came up with that name in the planning document back on July 6th. I guess `T` for Throwaway would work too.